### DISCUSSION TITLE
Best way to prevent user from accessing Contents/ContentItems

### DISCUSSION THREAD

**Discussion by jaypeemayo** [OrchardCMS/OrchardCore#10802] [Q&A] [answered]
Even for non admins , users can access this.

Example http://localhost:50854/Contents/ContentItems/4c3vf8ckmp40z54waerh9h2das

What is the best way to prevent the user from accessing this aside from blocking the request via route?

**Comment by deanmarcussen** [comment, marked as answer]
By removing the View Content Permission from the Anonymous role

**Reply by jaypeemayo** [reply]
thanks
